include/picongpu/fields/incidentField/profiles/DispersivePulse.def

```diff
@@ -55,6 +55,16 @@ namespace picongpu
                          */
                         static constexpr float_64 PULSE_INIT = 20.0;
 
+                        /** Width of the spectral support for the discrete Fourier transform of the pulse's field
+                         * from frequency-space domain to time-space domain.
+                         * Defined as number of sigmas of the laser pulse spectrum.
+                         * NOTE: A too small value will artificially increase the pulse duration and
+                         * too high values will increase the the time per step during initialization of the pulse.
+                         *
+                         * unit: none
+                         */
+                        static constexpr float_X SPECTRAL_SUPPORT = 4._X;
```

Comment:
I did originally think about doing this the SFINAE approach like in 

https://github.com/ComputationalRadiationPhysics/picongpu/blob/860f433600a956d8dd21236e53c550454d0199b9/include/picongpu/fields/incidentField/profiles/BaseParam.def#L163 

so that a user really doesn't have to touch it at all and it doesn't break any existing workflows.

Reply:
https://github.com/ComputationalRadiationPhysics/picongpu/blob/860f433600a956d8dd21236e53c550454d0199b9/include/picongpu/fields/incidentField/profiles/BaseParam.hpp#L99

Reply:
IMO the current user base is so small that we should enforce the value.